WindowManager/app/src/main/res/layout/activity_display_features.xml

```diff
@@ -24,110 +22,110 @@
     android:id="@+id/rootLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="com.example.windowmanagersample.DisplayFeaturesActivity">
+    tools:context="DisplayFeaturesActivity">
 
     <FrameLayout
-        android:id="@+id/feature_container_layout"
+        android:id="@+id/featureContainerLayout"
```

Comment:
Is this the new xml best format practice for naming?

Reply:
No, that's what I had in the code I took some of this from.

I've converted all the Camels to Snakes.